backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -176,19 +190,46 @@ protected Specification<Person> buildPersonMatchFilter(
    * @param facilityId If null, then it means across accessible facilities in the whole organization
    * @param pageOffset Pagination offset is zero based
    * @param pageSize How many results to return, zero will result in the default page size (large)
-   * @param isArchived Default is false. true will ONLY show deleted users
+   * @param includeArchivedPatients Default is false, true will ONLY show deleted users
    * @param namePrefixMatch Null returns all users, any string will filter by first,middle,last
    *     names that start with these characters. Case insenstive. If fewer than
    * @param includeArchivedFacilities setting to true will include patients in archived facilities,
    *     ignored if facilityId is not null
    * @return A list of matching patients.
    */
+  public List<Person> getPatients(
```

Comment:
maybe we could rename this getPatientsbyStatus or getPatientsbyArchiveStatus or something